lib/rspec.rb

```diff
@@ -9,7 +9,7 @@ def setup_rspec
   # folder for rspec
   run 'mkdir spec/codebase'
   run 'mkdir spec/fabricators'
-  run 'mkdir spec/features'
+  run 'mkdir spec/system'
```

Comment:
I don't like 'systems' but would adhere to conventions e.g. models, requests etc.

Reply:
@caherrerapa this is a new spec type introduced by Rails 5.1 so it's standard. We already had a discussion [here](https://github.com/nimbl3/rails-templates/pull/29#discussion_r151890678)